apps/microsoft-teams/app-actions/src/types.ts

```diff
@@ -82,6 +85,15 @@ export interface EntryActivity {
   entryUrl: string;
 }
 
```

Comment:
payload structure subject to change, pending an example payload coming from `workflow-consumer-api` service.